content/en/data_quality/_index.md

```diff
@@ -0,0 +1,180 @@
+        snowflake.execute({ sqlText: 'grant SELECT on all event tables' + schemaAndRoleSuffix});
+        snowflake.execute({ sqlText: 'grant SELECT on all external tables' + schemaAndRoleSuffix});
+        snowflake.execute({ sqlText: 'grant SELECT on all dynamic tables' + schemaAndRoleSuffix});
+        snowflake.execute({ sqlText: 'grant SELECT on future tables' + schemaAndRoleSuffix});
+        snowflake.execute({ sqlText: 'grant SELECT on future views' + schemaAndRoleSuffix});
+        snowflake.execute({ sqlText: 'grant SELECT on future event tables' + schemaAndRoleSuffix});
+        snowflake.execute({ sqlText: 'grant SELECT on future external tables' + schemaAndRoleSuffix});
+        snowflake.execute({ sqlText: 'grant SELECT on future dynamic tables' + schemaAndRoleSuffix});
+    }
+    
+    return 'Granted access to ' + numberOfSchemasGranted + ' schemas';
+    $$
+    ;
+
+    GRANT USAGE ON DATABASE IDENTIFIER($database_name) TO ROLE IDENTIFIER($role_name);
+    CALL grantFutureAccess('<DATABASE_NAME>', '<ROLE_NAME>');
+    ```
+
+1. (Optional) If your organization uses [Snowflake event tables][2], you can grant the Datadog role access to them.
+
+    ```sql
+    -- Grant usage on the database, schema, and table of the event table
+    GRANT USAGE ON DATABASE <EVENT_TABLE_DATABASE> TO ROLE IDENTIFIER($role_name);
+    GRANT USAGE ON SCHEMA <EVENT_TABLE_DATABASE>.<EVENT_TABLE_SCHEMA> TO ROLE IDENTIFIER($role_name);
+    GRANT SELECT ON TABLE <EVENT_TABLE_DATABASE>.<EVENT_TABLE_SCHEMA>.<EVENT_TABLE_NAME> TO ROLE IDENTIFIER($role_name);
+
+    -- Snowflake-provided application roles for event logs
+    GRANT APPLICATION ROLE SNOWFLAKE.EVENTS_VIEWER TO ROLE IDENTIFIER($role_name);
+    GRANT APPLICATION ROLE SNOWFLAKE.EVENTS_ADMIN TO ROLE IDENTIFIER($role_name);
+
+    ```
+
+After completing the Snowflake setup, configure the Snowflake integration in Datadog.
+
+1. On the [Snowflake integration tile][3], click **Add Snowflake account**.
+1. Enter your Snowflake account URL.
+1. Under **Logs**, turn on:
+   - **Query History Logs**
+   - **Enable Query Logs with Access History**
+1. Under **Data Observability**, turn on:
+   - **Enable Data Observability for Snowflake tables**
+1. Set the **User Name** to `DATADOG_USER`.
+1. Under **Configure a key pair authentication**, upload your unencrypted RSA private key.
+
+[1]: https://docs.snowflake.com/en/user-guide/key-pair-auth#generate-the-private-key
+[2]: https://docs.snowflake.com/en/developer-guide/logging-tracing/event-table-setting-up
+[3]: https://app.datadoghq.com/integrations?search=snowflake&integrationId=snowflake-web
+
+{{% /tab %}}
+{{% tab "BigQuery" %}}
+
+To monitor BigQuery data in Datadog, you must configure permissions in your Google Cloud project and enable the relevant features in the Datadog integration. For detailed instructions, see the [Expanded BigQuery monitoring][1] section of the Datadog Google Cloud Platform documentation.
```

Comment:
I don’t have confirmed steps for BigQuery setup, just the link to the doc I was given. Let me know if there are specific roles or toggles I should include, and I’ll update this section.